models/Recording.js

```diff
@@ -58,10 +60,18 @@ function canGetRaw() {
   return false;
 }
 
+function getFileName() {
+  var ext = "";
+  if (this.fileMimeType == 'video/mp4') ext = ".mp4"
+  return moment(new Date(this.recordingDateTime)).tz("Pacific/Auckland")
```

Comment:
What date type is `this.recordingDateTime`? Is it already a `Date` meaning there's no need to do the `new Date(...)`?

If it's a string, then I think `moment()` will parse it directly, so again there's no need for the `new Date(...)`.

Reply:
this.recordingDateTime is a string formatted to the ISO8601 standard.
So if `this.recordingDate` is null then `moment(this.recordingDate).format(...)` will return "Invalid". But with `moment(new Date(this.recordingDate)).format(...)` if `this.recordingDate` is null then it gets the date as 1970...

Reply:
Cool - makes sense. As long as it's there for a reason. 
Land away!